docs/fast-refresh.md

```diff
@@ -0,0 +1,38 @@
+---
+id: fast-refresh
+title: Fast Refresh
+---
+
+Fast Refresh is a React Native feature that allows you to get near-instant feedback for changes in your React components. Fast Refresh is enabled by default, and you can toggle "Enable Fast Refresh" in the [React Native developer menu](/intern/wiki/React_Native_Intro/common_workflows/#trigger-react-native-dev). With Fast Refresh enabled, most edits should fall within a second or two.
```

Comment:
"most edits should fall within a second or two." – I think "fall" is hard to understand for ESL folks. Maybe "Should take a second or two"?

Reply:
Changed to "most edits should be visible within a second or two"